.github/workflows/testing.yml

```
@@ -0,0 +1,63 @@
+name: GitHub Actions Linux Testing
+
+on:
+  push:
+    branches:
+    - master
+    - 'v1.*'
+  pull_request:
+  schedule:
+  - cron: 54 19 * * SUN # weekly at a "random" time
```

```suggestion
  - cron: '54 19 * * SUN' # weekly at a "random" time
```

I'm not convinced quotes makes a difference in this case, but GH docs say they're needed

```yaml
on:
  schedule:
    # * is a special character in YAML so you have to quote this string
    - cron:  '30 5,17 * * *'
```

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule

---

Looks like asterisks are for alias nodes. I only see the asterisk at the beginning of the string for those, and anyone running a cron Action every minute is probably being excessive. But yeah, let's put it in quotes.